### PR TITLE
Use HeatStackDomainAdminPassword during adoption

### PR DESCRIPTION
**Pull request by rabi**
We assume it's the same as service password which is not correct.

Depends-On: https://github.com/openstack-k8s-operators/openstack-operator/pull/1146

jira: https://issues.redhat.com/browse/OSPRH-10914

### CODE REVIEW

File: tests/secrets.sample.yaml [hunk 1]
```diff
@@ -21,9 +21,10 @@ ceilometer_password: "{{ lookup('file', tripleo_passwords) | from_yaml | communi
 cinder_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.CinderPassword') | first }}"
 glance_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.GlancePassword') | first }}"
 ironic_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.IronicPassword') | first }}"
-manila_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.ManilaPassword') | first }}"
+manila_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('      qword') | first }}"
```
Comment (bogdando) [inline tests/secrets.sample.yaml:24]: a typo?

**Comment by rabi** [comment]
recheck

**Comment by rabi** [comment]
Depends-On merged.

**Review by fao89 — APPROVED** [review]

**Pull request merged**